### PR TITLE
Added support back to iOS 9 by using alternative colours when below iOS 13

### PR DESCRIPTION
**Pull request by MarkBorazio**
When the project added in dark mode support, the minimum required version rose to iOS 13 because it used colours only on iOS 13 and above.
This pull request checks the iOS version and uses the old colours when below iOS 13. This means the minimum required iOS version has gone back down to iOS 9.

The colours used when below iOS 13 are the ones used before this commit: 7866158e6fb677ecadea766040ffe4dd317afbbc. Some of the colours that were changed in that commit are supported on iOS 9, so I left those as they are.

I have only tested this on iOS 12 and iOS 14. It would be prudent to test this on iOS 9 as well.

### CODE REVIEW

File: Classes/DoorbellDialog.m [hunk 1]
```diff
@@ -42,7 +42,11 @@ - (id)initWithFrame:(CGRect)frame
         boxFrame.size = CGSizeMake(300, 255);
         boxFrame = [self calculateNewBoxFrame:boxFrame];
         _boxView = [[UIView alloc] initWithFrame:boxFrame];
-        _boxView.backgroundColor = [UIColor systemBackgroundColor];
+        if (@available(iOS 13.0, *)) {
+            _boxView.backgroundColor = [UIColor systemBackgroundColor];
+        } else {
+            _boxView.backgroundColor = [UIColor whiteColor];
+        }
```
Comment (MarkBorazio) [inline Classes/DoorbellDialog.m:49]: I considered putting this in a new `Colours` file, but the project has a very simple project structure, so I decided to just place it here. It does mean that we have a few annoying `if else` statements in this file.

File: Classes/DoorbellDialog.m [hunk 2]
```diff
@@ -296,8 +300,16 @@ - (void)createBoxSubviews
...
+    if (@available(iOS 13.0, *)) {
+        _bodyView.backgroundColor = [UIColor systemBackgroundColor];
+    } else {
+        _bodyView.backgroundColor = [UIColor whiteColor];
+    }
+    if (@available(iOS 13.0, *)) {
+        //    _bodyView.textColor = [UIColor darkTextColor];
+    } else {
+        _bodyView.textColor = [UIColor darkTextColor];
+    }
```
Comment (MarkBorazio) [inline Classes/DoorbellDialog.m:312]: This was only commented out once dark mode support was added, so I tried to keep things the same.

File: Classes/DoorbellDialog.m (hunk 1 above)
Comment (manavo) [reply Classes/DoorbellDialog.m:49]: Definitely happy keeping the simpler structure!

File: Classes/DoorbellDialog.m (hunk 2 above)
Comment (manavo) [reply Classes/DoorbellDialog.m:312]: Yep thanks! Needed to comment it out, otherwise the text color stayed dark even in dark mode and could not be read!

**Pull request merged**